crest/Assets/Crest/Crest/Shaders/OceanInputs/AnimWavesSetHeightToGeometry.shader

```diff
@@ -4,7 +4,7 @@
 
 // This writes straight into the displacement texture and sets the water height to the y value of the geometry.
 
-Shader "Crest/Inputs/Animated Waves/Set Water Height To Geometry"
+Shader "Crest/Inputs/Animated Waves/Lock Water Height To Geometry"
```

Comment:
If we go with the other change, we could keep this as "Set". I am thinking of changing "Override" to "Set" for foam and shadows too since I am thinking that it was a poor choice on my part.

Reply:
done but with slight change noted below

Reply:
oh and yeah think i agree on the other shader changes, feel free to throw them in

Reply:
Nice. I'll do a separate PR.